{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "b6ec5f82_a969bfb8",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1025452
      },
      "writtenOn": "2021-01-13T10:03:18Z",
      "side": 1,
      "message": "Verified by building locally the commit in question",
      "revId": "5f9834ecf12604b35b8e528fb66a8f120671dd94",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "b7add5f0_704dac30",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1012732
      },
      "writtenOn": "2021-01-13T21:31:33Z",
      "side": 1,
      "message": "\u003e Verified by building locally the commit in question\n\nStand-alone I guess; alright, thanks! :)",
      "parentUuid": "b6ec5f82_a969bfb8",
      "revId": "5f9834ecf12604b35b8e528fb66a8f120671dd94",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "dff7d573_18c19c25",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1025452
      },
      "writtenOn": "2021-01-14T08:17:37Z",
      "side": 1,
      "message": "Right, I have also called\n\nbazel test //...\n\npost-factum and it fails :/ but it also fails with c6a12d203e795b269ce06b62faef8973c6689339 (parent) after calling\n\nbazel clean --expunge\n\nso I don\u0027t believe that this upgrade is a culprit here... should we go ahead and try to fix those tests in the follow up change?",
      "parentUuid": "b7add5f0_704dac30",
      "revId": "5f9834ecf12604b35b8e528fb66a8f120671dd94",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "a7deab53_bc21deca",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1012732
      },
      "writtenOn": "2021-01-14T19:25:12Z",
      "side": 1,
      "message": "I\u0027m able to pass plugin\u0027s tests using stand-alone locally. Depends on which exact failures or error(s) you\u0027re getting..",
      "parentUuid": "dff7d573_18c19c25",
      "revId": "5f9834ecf12604b35b8e528fb66a8f120671dd94",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "4720f155_65dd2aae",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1025452
      },
      "writtenOn": "2021-01-15T09:19:56Z",
      "side": 1,
      "message": "I have figured out that it is about JVM - forgot to switch to Java8 before calling bazel and it failed ;)",
      "parentUuid": "a7deab53_bc21deca",
      "revId": "5f9834ecf12604b35b8e528fb66a8f120671dd94",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    }
  ]
}